pkg/asset/installconfig/powervs/session.go

```diff
@@ -85,3 +87,19 @@ func getPISession() (*ibmpisession.IBMPISession, error) {
     // @TODO: pass through debug?
 	return ibmpisession.New(passwd,region, false, defSessionTimeout, id, zone)
 }
+
+func getUserCredentials() error {
+  var keyID string
+  err := survey.Ask([]*survey.Question{
+    {
+      Prompt: &survey.Input{
```

Comment:
would prefer that this come from a json file from a well defined default path like `{$HOME}/.bluemix` or something. asking for the api key means making it visible in the install-config which is not a great idea.

Reply:
Yah I think that's fine, if no other providers do this. I've started doing some work locally to read from ~/.bluemix. The only negative is that this would require that users have the ibmcloud cli installed. There's also a possibility that userid and apikey are set as env vars (a precedent described here: https://github.com/IBM-Cloud/power-go-client#ibm-cloud-sdk-for-power-cloud). Either way, though, this will be needed by TF for us to create several of the resources, so it will end up in the tfvars.

Reply:
i don't think having ibmcloud cli is so bad. i mean for aws, azure etc...we install the corresponding tools anyway, so it's in line with those

Reply:
Looking at aws, correct me if I am wrong, but they seem to handle their "secret keys" by prompting the user for them, similar to how we prompt the user for the pull secret, which will pop out into the install-config.  Further I believe that aws will pull these "secret keys" from the current session if they are already defined.  I still need to track down the two variables we use, and force a prompt if they are not defined.
export IBMID=$IAAS_CLASSIC_USERNAME
export IBMID_PASSWORD=$clear_text_password

Here's a snip from aws session.go for reference to save having to dig for it
var secretKey string
        err = survey.Ask([]*survey.Question{
                {
                        Prompt: &survey.Password{
                                Message: "AWS Secret Access Key",
                                Help:    "The AWS secret access key corresponding to your access key ID (this is not your password).",
                        },
                },
        }, &secretKey)
        if err != nil {
                return err
        }

Reply:
ah right ..yes aws does this...ok..i guess it's fine to do this.